android/app/src/main/res/menu/bottom_nav_menu.xml

```diff
@@ -2,15 +2,15 @@
 <menu xmlns:android="http://schemas.android.com/apk/res/android">
 
     <item
-        android:id="@+id/itemHome"
+        android:id="@+id/mainFragment"
```

Comment:
This is not a blocker but i didn't understand why did you change the ids of these items? These namings do not fit the convention we decided.

Reply:
AFAIK, it needs to match the fragment ids in the `main_navigation.xml` to navigate properly.

Reply:
okey, thanks for explanation.